### ISSUE TITLE
Jenkins marian-dev-cpu-avx512 #122 failed

### ISSUE THREAD

**Issue by marianminion** [marian-nmt/marian-dev#855]
Build 'marian-dev-cpu-avx512' is failing!

Last 50 lines of build output:

```
[...truncated 50.13 KB...]
[04/15/2021 10:37:57] Test took 00:00:12.867s
[04/15/2021 10:37:57] Checking directory: tests/decoder/wmt16
[04/15/2021 10:37:57] Running setup script
[04/15/2021 10:37:57] Running tests/decoder/wmt16/test_ende_cpu.sh ... OK
[04/15/2021 10:39:29] Test took 00:01:31.971s
[04/15/2021 10:39:29] Checking directory: tests/models/transformer
[04/15/2021 10:39:29] Running setup script
[04/15/2021 10:39:29] Running tests/models/transformer/test_hard_aligns_cpu.sh ... OK
[04/15/2021 10:39:48] Test took 00:00:19.138s
[04/15/2021 10:39:48] Checking directory: tests/models/wngt19
[04/15/2021 10:39:48] Running setup script
[04/15/2021 10:39:48] Running tests/models/wngt19/test_model_base_fbgemm_packed16.sh ... OK
[04/15/2021 10:40:07] Test took 00:00:18.348s
[04/15/2021 10:40:07] Running tests/models/wngt19/test_model_base_fbgemm_packed8.sh ... OK
[04/15/2021 10:40:22] Test took 00:00:15.321s
[04/15/2021 10:40:22] Checking directory: tests/models/wnmt18
[04/15/2021 10:40:22] Running setup script
[04/15/2021 10:40:22] Running tests/models/wnmt18/test_student_small.sh ... OK
[04/15/2021 10:40:39] Test took 00:00:16.557s
[04/15/2021 10:40:39] Running tests/models/wnmt18/test_student_small_aan.sh ... failed
[04/15/2021 10:40:47] Test took 00:00:8.220s
[04/15/2021 10:40:47] Running tests/models/wnmt18/test_student_small_aan_intgemm16.sh ... failed
[04/15/2021 10:40:55] Test took 00:00:8.073s
[04/15/2021 10:40:55] Running tests/models/wnmt18/test_student_small_aan_intgemm8.sh ... failed
[04/15/2021 10:41:03] Test took 00:00:7.900s
[04/15/2021 10:41:03] Checking directory: tests/scorer/scores
[04/15/2021 10:41:03] Running setup script
[04/15/2021 10:41:03] Running tests/scorer/scores/test_scores_cpu.sh ... OK
[04/15/2021 10:41:41] Test took 00:00:37.485s
[04/15/2021 10:41:41] Checking directory: tests/server
[04/15/2021 10:41:41] Running setup script
[04/15/2021 10:41:42] Running tests/server/test_ende_cpu.sh ... OK
[04/15/2021 10:42:23] Test took 00:00:41.247s
[04/15/2021 10:42:23] Checking directory: tests/training/restoring/multi-gpu
[04/15/2021 10:42:23] Running setup script
[04/15/2021 10:42:23] Running tests/training/restoring/multi-gpu/test_adam_sync_cpu.sh ... OK
[04/15/2021 10:43:06] Test took 00:00:42.293s
---------------------
Failed:
  - tests/models/wnmt18/test_student_small_aan.sh
  - tests/models/wnmt18/test_student_small_aan_intgemm16.sh
  - tests/models/wnmt18/test_student_small_aan_intgemm8.sh
Logs:
  - /var/lib/jenkins/workspace/marian-dev-cpu-avx512/regression-tests/tests/models/wnmt18/test_student_small_aan.sh.log
  - /var/lib/jenkins/workspace/marian-dev-cpu-avx512/regression-tests/tests/models/wnmt18/test_student_small_aan_intgemm16.sh.log
  - /var/lib/jenkins/workspace/marian-dev-cpu-avx512/regression-tests/tests/models/wnmt18/test_student_small_aan_intgemm8.sh.log
---------------------
Ran 19 tests in 00:06:31.118s, 16 passed, 0 skipped, 3 failed
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [hihoan] 96ed0baf5a8a0c39508fb53ff13e0ab309fd7cea - chmod -x

- [hihoan] 0d8372c590b290dc9b10200ea068c407954229a8 - move embedding to its own file

- [hihoan] ca47eabca5cb9bb11a3e4fe45afa77501128b4b9 - move output to its own file

- [hihoan] f7266886f0d478d802a88f2ce82b71f27c37bf07 - move logits to its own file

- [hihoan] 42406cc715a301db2c3087d0def6ff70332d1074 - move logits to its own file

- [hihoan] b88c3fcb71ad06326f60cc930fd283d77cd41a6c - costs.cpp

- [hihoan] 085c8a7a98a462ea809dc4efcff7ba0f5b49066d - more code from .h -> .cpp

- [hihoan] 7c1cb8462a2adc6540ee78f111a75ed4fbdd66ad - add logits.cpp

- [hihoan] 55f4216552bca148091f15b72c5c2e5b486d4c79 - add .h

- [hihoan] ba196637847c50c76d5d0edfcfe39b9cedb0d1d0 - clang-format -i

- [Roman Grundkiewicz] cd018e8d0404687c0bd13f64962bd22617b80331 - Update formatting

- [github] 77c3e356a47113f661dda794b815e84561ca93f5 - Install Boost in GitHub workflows (#834)

- [Roman Grundkiewicz] a1aaa32c6af15f9bc653b6ceeb81ac1767ef3f39 - Merged PR 18201: Install Boost in Azure pipelines

- [Marcin.JunczysDowmunt] e08c52a8df4be5cf2d21cbb6e1eb3a5dcff0d0e2 - Merged PR 18185: Support for Microsoft legacy binary shortlist

- [Marcin Junczys-Dowmunt] 8f73923d3134f4799497b7e880963336b8fe4d6b - increase version and update changelog

- [github] 571634a2c05b9bc2de206b1061fbc39a6e97087a - Update GitHub workflows with Ubuntu+CUDA (#837)

- [Roman Grundkiewicz] 326b9400e9b4dfa6bbf445e4cbbc83ea5145b6b7 - Merged PR 18232: Update VS CMake builds and scripts

- [github] db2a5e4d66e92dbed7711ace8393f3fd618b89f4 - Fix broken links to MNIST data sets (#838)

- [github] a11418c17c5e295ddf5bfd7b9963cfec78e3852a - Add simple unit tests for binary files (#826)

- [github] c724837ab326ceab629e30f7a7ae89e45065f3df - Update VERSION

- [github] d780082973f7de03c9426a5b2fa7936408282114 - Fix model loading on architectures where size_t is 32bits (#825)

- [github] c89efbe9192974664ef8a54b99f2202c0ac12411 - Update VERSION

- [github] 096c48e51cd2e61bb275345d7cca99cbfd6bc5c7 - remove TC_MALLOC from optional dependencies (#840)

- [github] 0394d2cdbe42b7ad7fc02854da8e063f2a23496f - Display decoder speed statistics with --stat-freq N (#841)

- [github] b36d0bbbab1fa4d435e517c974c3fde96e3145fe - Fix FBGEMM build with gcc 9.3+ (#836)

- [hihoan] 415769fb2f953c5abf88f2d3498f2a46ea3607d7 - start lsh shortlist

- [hihoan] 64707fa484c79fe68fdf5022b269b99d70bee5fd - Revert "start lsh shortlist"

- [github] ffd997e36015e1022a122095ebb8e147c5b59650 - Properly copy the entire vector in the int16_t case (#845)

- [Marcin.JunczysDowmunt] 7d1f941242928c976640a20f37e1bd9ac10011e8 - Merged PR 18309: Cleaner suppression of unwanted output words

- [Roman Grundkiewicz] c29cc83dc4d234f0e3a00a46a729053132b408b8 - Update submodule examples

- [Marcin.JunczysDowmunt] 4408e88a941b60e23ddb12e3afcf599ec0f42e9b - Merged PR 18366: Fix generation of special control characters for default vocabulary

- [Marcin Junczys-Dowmunt] bfa6180033307f579f91d4220971474d0c3de86d - Revert "remove TC_MALLOC from optional dependencies (#840)"

- [github] 0223ce90b1d6afaa047f2baeb4d0689f87d7ae81 - Fix Ubuntu GitHub checks (#848)

- [github] fddd0e0661eb13b2a132e401e268315a35e468f7 - Adds better Affine support for GPUs when using CUDA 11. Introduces a new bias addition kernel for CUDA < 11 (#778)

- [github] a17ee300f4567cb747a93b0559d48b664d0e882e - Create VERSION

- [github] be6506562321be49f8ef4b8c2a469662bbc1cfa4 - Allow to choose fine-grained CPU intrinsics on as CMake options (#849)

- [github] fdf9fe7d4a98268b5d451bc5b5944da959fbe8e2 - Update VERSION

- [Marcin.JunczysDowmunt] a05124176d8869962b717a3557c383406f8c76f4 - Merged PR 18531: Install GCC in Azure pipelines

- [Marcin.JunczysDowmunt] caddad90cdb06283da2f5d17a3340ca8c6387b38 - Merged PR 18505: RMSNorm on GPU

- [github] a7c3a0b2ef0d3a9ffd952bb1eb97e680633d1515 - fix beam_search ABORT when enable openmp and OMP_NUM_THREADS > 1 (#767)

- [Marcin Junczys-Dowmunt] 8a53b761d5bc922e4ab058a4487ad362d2edefaf - update version

- [github] bb6092da2b89c3882aa256480cef13081fd6f50f - Compute tensor size using integers (#851)

[View full output](http://vali.inf.ed.ac.uk/jenkins/job/marian-dev-cpu-avx512/122/)

**Comment by marianminion** [comment]
Build was fixed!
